### PR TITLE
Changing Top Radio Buttons to Buttons and other changes

### PR DESCRIPTION
**Pull request by stevenndungu**
@africanmathsinitiative/developers this is ready for review
-Changed top radio buttons to buttons similar to those in frequency tables in Prepare Menu dialogues 
-Fixed a developer errors on Add comment dialogue and disabled it for now that its method is not yet ready.
-Small but important edits on  View Graph and View Label Factors dialogues - word "label" was not visible and set the  label on the selector to be more specific.

### CODE REVIEW

File: instat/dlgUnusedLevels.vb [hunk 1]
```diff
@@ -31,6 +31,7 @@ Public Class dlgUnusedLevels
         SetRCodeforControls(bReset)
         bReset = False
         autoTranslate(Me)
+        TestOKEnabled()
     End Sub
```
Comment (stevenndungu) [inline instat/dlgUnusedLevels.vb]: This dialogue on first load TestOk was enabled with the receiver empty

**Comment by lilyclements** [comment]
I'm going to review this now, and potentially make changes as I go!

**Comment by lilyclements** [comment]
Made a few changes which are waiting to be merged in. I was just wondering why Insert Columns hasn't been updated yet? @maxwellfundi it says you were working on this on the excel doc. What was the issue that made work stop on this dialog?

**Comment by dannyparsons** [comment]
Insert Columns sould be a job for someone looking for a new task @stevekogo @AlexSananka? Finding old dialogs to update from the Excel sheet is something everyone could do when they need tasks.

**Comment by lilyclements** [comment]
@dannyparsons this is ready to review

**Pull request merged**